.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: PHPUnit
+
+on: [push, pull_request]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest, windows-latest, macos-latest]
+        php-versions: ['7.2', '7.3', '7.4']
+    name: "PHP ${{ matrix.php-versions }} test on ${{ matrix.operating-system }}"
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          tools: composer
+
+      - name: Setup problem matchers for PHPUnit
+        run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Install dependencies
+        run: composer install --no-progress --prefer-dist --optimize-autoloader
+
+      - name: Test with phpunit
+        run: vendor/bin/phpunit --coverage-text
```

Comment:
Do we want to use `phpunit.xml.dist`? Or should we keep that file if not?

Reply:
It will automatically use the file. I don't recall the history behind explicitly specifying it. Perhaps it was required as some point or it was just cargo-culted from somewhere else.